### ISSUE TITLE
Microsoft.Framework.WebEncoders.UnicodeRangesTests.Range_All failed in Windows CI build

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#1887]
Failed in Linux CI build:
```
Starting:    System.Text.Encodings.Web.Tests
     Microsoft.Framework.WebEncoders.UnicodeRangesTests.Range_All [FAIL]
        Assert.Same() Failure
        Expected: UnicodeRange { FirstCodePoint = 0, Length = 65536 }
        Actual:   UnicodeRange { FirstCodePoint = 0, Length = 65536 }
        Stack Trace:
d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Text.Encodings.Web\tests\UnicodeRangesTests.cs(209,0): at Microsoft.Framework.WebEncoders.UnicodeRangesTests.Range_Unicode(UInt16 first, UInt16 last, String blockName)
d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Text.Encodings.Web\tests\UnicodeRangesTests.cs(32,0): at Microsoft.Framework.WebEncoders.UnicodeRangesTests.Range_All()
```

**Comment by stephentoub** [comment]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_windows_debug_prtest/1508/console

**Comment by KrzysztofCwalina** [comment]
@stephentoub, I created a PR to fix that: https://github.com/dotnet/corefx/pull/1889

**Comment by stephentoub** [comment]
Fixed by #1889
